### PR TITLE
Add knobs to storybook #11

### PR DESCRIPTION
**Pull request by nfiniteset**
Add knobs to storybook #11

### CODE REVIEW

**Comment by camwest** [comment]
@nfiniteset can you show an example of them being used in this PR?

**Comment by nfiniteset** [comment]
Yeah, working on it. Was just getting a WIP PR up.

**Comment by nfiniteset** [comment]
This is still WIP. The example fails due to `template is undefined` error in `styles.js`.

File: packages/storybook/.storybook/config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { configure } from '@kadira/storybook';
 
 function loadStories() {
-  require('../stories/index.js');
```
Comment (camwest) [inline packages/storybook/.storybook/config.js]: FYI I turned this requirement off: https://github.com/orion-ui/orion/blob/67de5837d47eafafd6a02578d8dfa3d2c0be0075/packages/eslint-config-orion/index.js#L11

**Comment by nfiniteset** [comment]
I renamed index.jsx to index.js. Anything else here before we merge it?

**Pull request merged**